config/locales/en.yml

```diff
@@ -1724,9 +1729,16 @@ en:
       wrong_user: "You are logged in as `%{user}' but the message you have asked to read was not sent by or to that user. Please login as the correct user in order to read it."
     sent_message_summary:
       destroy_button: "Delete"
+    heading: 
+      my_inbox: "My Inbox"
+      my_outbox: "My Outbox"
+      muted_messages: "Muted messages"
```

Comment:
Should the capitalisation be consistent here given these are three labels that sit adjacent to each other?

Reply:
To me the 3 label terms here are different. My reasoning:

- `Inbox` and `Outbox` are defined terms and therefore should be capitalised.
- `Muted messages` is not.

Bear in mind that I am not a native english speaker so I'm happy to adjust the label to `Muted Messages if you say so.